### PR TITLE
Shows fatal error for invalid help text requests

### LINKED ISSUE #7444: Content spoofing, action=helpadmin;help=payload
Admin help popup apparently allows for easy content spoofing. 
Reported here https://www.simplemachines.org/community/index.php?topic=582136
Confirmed to work on simplemachines.org https://www.simplemachines.org/community/index.php?action=helpadmin;help=Content%20Spoofing%20Bug

### PR DESCRIPTION
**Pull request by Sesquipedalian**
Fixes #7444

### CODE REVIEW

File: Sources/Help.php
Comment (live627) [inline Sources/Help.php:118]: This might make the popup look funky. I'll need to test it to confirm.

Comment (Sesquipedalian) [reply Sources/Help.php:118]: It just makes the popup show the string "error" in its content area.

Comment (live627) [reply Sources/Help.php:118]: I guess that could work

Comment (Sesquipedalian) [reply Sources/Help.php:118]: Yeah. The main concern is people accessing the URL directly rather than via the popup.

**Review by live627 — APPROVED** [review]

**Comment by live627** [comment]
@LexArma @sbulen I can merge this once you two approve this 😀

**Review by LexArma — APPROVED** [review]
Looks good to me, basically as long as it doesn't accept direct user input, I'm fine.

**Pull request merged**
